contrib/google.golang.org/grpc/server.go

```diff
@@ -75,15 +76,32 @@ func StreamServerInterceptor(opts ...Option) grpc.StreamServerInterceptor {
 
 		// if we've enabled call tracing, create a span
 		if cfg.traceStreamCalls {
+			var methodKind string
+			if info != nil {
```

Comment:
I don't think `info` will ever be `nil`. Let's not overdo the `nil` checking. The most extreme case of `nil` checking is checking method receivers too :)

Do you have any reason to believe this will ever be `nil`?

Reply:
I don't. I looked at all calling functions when I added the nil check, and it's never nil. I only added the check in an abundance of caution. I will remove it.